lib/streamlit/MediaFileManager.py

```diff
@@ -16,11 +16,18 @@
 """Provides global MediaFileManager object as `media_file_manager`."""
 
 import hashlib
-from datetime import datetime
+import collections
+
+from streamlit.ReportThread import get_report_ctx as _get_report_ctx
```

Comment:
Remove the `as _get_report_ctx`

Reply:
I was just following patterns I saw elsewhere.